third_party/mpi/BUILD

```diff
@@ -0,0 +1,24 @@
+licenses(["restricted"])
+
+filegroup(
+    name = "all_files",
+    srcs = glob(
+        ["**/*"],
+        exclude = [
+            "**/METADATA",
+            "**/OWNERS",
+        ],
+    ),
+    visibility = ["//tensorflow:__subpackages__"],
+)
+
+load("//third_party/mpi:mpi.bzl", "mpi_hdr")
+
+cc_library(
```

Comment:
This target must be buildable even if MPI is disabled, because `bazel test ...` will try to build all targets. See `third_party/mkl` for an example of how to do that.